fastchat/conversation.py

```diff
@@ -840,6 +840,18 @@ def get_conv_template(name: str) -> Conversation:
     )
 )
 
+# Mistral template
```

Comment:
If possible, add a link to the source

Reply:
done (https://docs.mistral.ai/llm/mistral-instruct-v0.1#chat-template)